infra/README.md

```diff
@@ -8,6 +8,13 @@
 
 ## Rebuilding `makemake`
 
+### Applying patches automatically
+By default, a [GitHub
+action](https://github.com/ngi-nix/ngipkgs/actions/workflows/makemake.yaml)
+deploys `makemake` automatically on every merge. No manual actions are required
```

Comment:
```suggestion
deploys `makemake` automatically (on every push to main). No manual actions are required
```

Reply:
@lorenzleutgeb we already discussed this, @Erethon will reformat to one sentence per line (and maybe add a note in the contributing guide).

Reply:
I can't find the discussion, sorry.

Reply:
The discussion happened on a call last week, don't worry about it.